packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -27,8 +26,21 @@ if (process.env.TEST_FILE) {
   tests = specFiles.map(file => file.file)
 }
 
+const isOrbitTest = process.env.E2E_ORBIT == 'true'
 const shouldRecordVideo = process.env.CYPRESS_RECORD_VIDEO === 'true'
 
+const l1WethGateway = isOrbitTest
```

Comment:
weth addresses moved here because in the other file we don't have the context of `process.env.E2E_ORBIT`. Now we pass these addresses as Cypress envs